recipes/discorpy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: discorpy
+  version: 1.4
+
+source:
+  url: https://github.com/DiamondLightSource/discorpy/archive/refs/tags/v1.4.tar.gz
+  sha256: 4144dcad781312c8b2316d785c9e8110715b2fbc7f15180e1760b9191f0eb9c7
+build:  
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+  run:
+    - python >=3.7
+    - matplotlib
+    - numpy
+    - scipy
+    - h5py
+    - pillow
+    - scikit-image
+
+test:
+  imports:
+    - discorpy
+  source_files:
+    - ./tests/
+  commands:
+    - python -m unittest discover -s tests -v
+
+about:
+  home: https://github.com/DiamondLightSource/discorpy
+  license:  Apache-2.0
+  license_file: LICENSE
+  summary: 'Distortion correction with high-accuracy for lens-coupled detector systems: https://discorpy.readthedocs.io'
+
+extra:
+  recipe-maintainers:
+    - nghia-vo
```

Comment:
It's fine to include them, but you also need to list yourself since you submitted the recipe.